be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -188,10 +190,17 @@ Status StreamLoadExecutor::commit_txn(StreamLoadContext* ctx) {
     TNetworkAddress master_addr = get_master_address();
     TLoadTxnCommitResult result;
 #ifndef BE_TEST
-    RETURN_IF_ERROR(ThriftRpcHelper::rpc<FrontendServiceClient>(
+    auto st = ThriftRpcHelper::rpc<FrontendServiceClient>(
             master_addr.hostname, master_addr.port,
             [&request, &result](FrontendServiceConnection& client) { client->loadTxnCommit(result, request); },
-            config::txn_commit_rpc_timeout_ms));
+            config::txn_commit_rpc_timeout_ms);
+    if (st.is_thrift_rpc_error()) {
+        return Status::ServiceUnavailable(fmt::format(
```

Comment:
What's the purpose to change the error type from `ThriftRPCError` to `ServiceUnavailable`?

Reply:
convenient for user to distinguish between errors